src/AtlassianAssistance.JiraService/Attributes/InsightAttributeFieldAttribute.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace AtlassianAssistance.JiraService.Attributes
+{
+    public sealed class InsightAttributeFieldAttribute : Attribute
```

Comment:
Please add a notation to use just in properties.

Reply:
[94dfb5e](https://github.com/kalhorim/JiraService/commit/94dfb5ec1c0052e78cdb631c9081becb8f45b00a)